webapp/templates/partials/scoreboard_table.html.twig

```diff
@@ -106,7 +106,7 @@
     <tbody>
     {% set previousSortOrder = -1 %}
     {% set previousTeam = null %}
-    {% set backgroundColors = {"#FFFFFF": 1} %}
+    {% set backgroundColors = {"#FFFFFF": 1, '#FFFFE0': 1} %}
```

Comment:
What's this for? Is it still needed after resolving the previous comment? (also on line 359)

Reply:
I do think we don't have to do the loop twice though. let me fix that